CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## Supported Release 1.5.0
+### Summary
+
+A release which has several support additions for different OSes. Also a couple of additional features and a few bug fixes.
+
+#### Features
+- Added Ubuntu 15.10 support.
```

Comment:
Instead of "support" can you change that to "compatibility". Since 15.10 won't be an "officially supported" release for this module.